packages/flutter/lib/src/material/search.dart

```diff
@@ -344,6 +344,15 @@ abstract class SearchDelegate<T> {
       ..pop(result);
   }
 
+  // Closes the search page and returns to the underlying route whitout result.
```

Comment:
Public docs start at `///` per style guide.
```suggestion
  /// Closes the search page and returns to the underlying route whitout result.
```

(Great name!)